template.js

```diff
@@ -0,0 +1,66 @@
+const readline = require('readline');
+
+const readlineInterface = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+function ask(questionText) {
+  return new Promise((resolve, reject) => {
+    readlineInterface.question(questionText, resolve);
+  });
+};
+
+let player = 'X'
+
+let squares = ['1', '2', '3', '4', '5', '6', '7', '8', '9']
+
+function setUp() {
+  player = 'X';
+  squares = ['1', '2', '3', '4', '5', '6', '7', '8', '9'];
+
+  play()
+}
+
+function toggle() {
+  if(player === 'X') {
+    player = 'O'
+  }
+  else{ player = 'X' }
+}
+
+function checkWin() {
+  if ((squares[0] === player && squares[1] === player && squares[2] === player) ||
```

Comment:
think about how to write this by looping over a data structure instead of repeating `(squares[ ] === player && squares[ ] === player && squares[ ] === player)` a dozen times